JS/Lesson15/sinan/script.js

```diff
@@ -175,5 +317,32 @@ whichMonthName(10);
 whichMonthName(2);
 
 // 14. Write a program that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). Use a switch statement to handle the different cases.
+function gradeTest (point){
+	switch(point){
```

Comment:
It's a good practice to always make sure that the value you're using is in the correct format. YOu should do that inside the function instead of relying on the value to be passed in correct format from outside.
```suggestion
	switch(point.toUpperCase()){
```